packages/orchestrator-ui-components/src/components/WfoBadges/WfoLibraryVersionBadge/WfoLibraryVersionBadge.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { EuiText } from '@elastic/eui';
+
+import { ORCHESTRATOR_UI_LIBRARY_VERSION } from '@/configuration';
+
+import { useOrchestratorTheme } from '../../../hooks/useOrchestratorTheme';
+
+export const WfoLibraryVersionBadge = () => {
+    const { theme } = useOrchestratorTheme();
+
+    return (
+        <EuiText
+            color={theme.colors.ghost}
+            size="xs"
+            style={{ display: 'inline-block', marginLeft: theme.base }}
+        >
+            v{ORCHESTRATOR_UI_LIBRARY_VERSION}
```

Comment:
Nitpick but I'm not sure about the v here. I think we could do without @wouter1975 ? 

```suggestion
            {ORCHESTRATOR_UI_LIBRARY_VERSION}
```

Reply:
It was without "v" before, but @wouter1975 asked me to add it